tests/unit/algo/classification/test_otx_dino_v2.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+from otx.algo.classification import DINOv2RegisterClassifier
+from otx.core.data.entity.base import OTXBatchLossEntity
+from otx.core.data.entity.classification import MulticlassClsBatchDataEntity, MulticlassClsBatchPredEntity
+from torch import nn
+
+if TYPE_CHECKING:
+    from omegaconf import DictConfig
+
+class TestDINOv2RegisterClassifier:
+    def test__init(self, fxt_config_mock: DictConfig) -> None:
```

Comment:
What's the reason behind this double underscore?

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2708/commits/7c8db3a54e84a4e183a48378fdd428e047288441